### ISSUE TITLE
used python 3.7.15 but still having problems installing requirements.txt

### ISSUE THREAD

**Issue by KLAU-S** [mftnakrsu/Automatic_Number_Plate_Recognition_YOLO_OCR#3]
![image](https://user-images.githubusercontent.com/75158477/206722934-c550d84a-51fd-452c-ae3d-267f1cc1edda.png)

**Comment by mftnakrsu** [comment]
Have you tried creating an anaconda virtual environment?  conda create -n yourenvname python=x.x anaconda
Also, the error you get says your python version must be greater than 3.8? Have you tried py3.8?

**Comment by KLAU-S** [comment]
yeah, created conda environment and used 3.8.15 now requirements.txt seems to be installed but getting this error now ! I am sorry if i am bothering you too much, but only you can help !
![image](https://user-images.githubusercontent.com/75158477/206826943-063d1360-13a0-47ae-b5e4-cd5d21ca89d3.png)

**Comment by KLAU-S** [comment]
did you hardcode this urlpath somewhere or i don't know, (\\home\\mef\\Documents\\plate_detection_project\\best.pt)
I have wasted so much time in this !

**Comment by mftnakrsu** [comment]
yes, in helper/params.py .

**Comment by KLAU-S** [comment]
bro, can i ask 1 thing, if you can give 10-20 min of your time. its fine if not today, maybe tomorrow. But please google meet with me, just make me run your repo in my system and done. please please please please please.

**Comment by ldaus** [comment]
I have the same issue with `\\home\\mef\\Documents\\plate_detection_project\\best.pt` but the directory didn't change if i change it in `helps/params.py.`  Any further information, where i have to setup the model?
